core/storage/classifier/gae_models.py

```diff
@@ -111,3 +116,87 @@ def create(
 
         classifier_model_instance.put()
         return instance_id
+
+
+class TrainClassifierJobModel(base_models.BaseModel):
+    """Model for storing classifier training jobs.
+
+    The id of instances of this class has the form
+    {{exp_id}}.{{random_hash_of_16_chars}}
+    """
+
+    # The ID of the algorithm used to create the model.
+    algorithm_id = ndb.StringProperty(required=True, choices=ALGORITHM_CHOICES)
```

Comment:
See https://cloud.google.com/appengine/docs/standard/python/ndb/entity-property-reference#properties_and_value_types : "Properties can be indexed or unindexed (queries that order or filter on a property P will ignore entities where P is unindexed)".

So if you don't index a property then it's not going to appear in  "SELECT entries where property X has value Y"-type queries.